src/HomeWork7/Cosmos/SpacePort.java

```diff
@@ -0,0 +1,79 @@
+package HomeWork7.Cosmos;
+
+public class SpacePort {
+    private boolean check;
+    private IStart iStart;
+    public void setCheck(boolean b){
+        this.check = b;
+    }
+    public void start(IStart start){
+        if (start.checkSystem(true) == check){
+            System.out.println("система в порядке горовтесь к старту");
+            Shuttle shuttle = new Shuttle();
```

Comment:
we create object in main class and send it in this method. for example in main:
IStart shuttle = new Shuttle();
start(shuttle);